<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>size</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="sizeThumbnailButton">
        <property name="text">
         <string>thumbnail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sizeSmallButton">
        <property name="text">
         <string>small</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sizeMidButton">
        <property name="text">
         <string>mid</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sizeBigButton">
        <property name="text">
         <string>big</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>name</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="idRadioButton">
        <property name="text">
         <string>id</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="titleRadioButton">
        <property name="text">
         <string>title</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="idTitleRadioButton">
        <property name="text">
         <string>id_title</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QListWidget" name="photosListWidget"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Code:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="flickrLabel">
     <property name="text">
      <string notr="true">&lt;a href=&quot;http://picasaweb.google.com&quot; title=&quot;Picasa&quot;&gt;&lt;img src=&quot;:/plugins/picasa-download/picasa&quot; width=&quot;32&quot; height=&quot;32&quot; alt=&quot;&quot;&gt;&lt;/a&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="activeButton">
     <property name="text">
      <string>activate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="codeLineEdit">
     <property name="toolTip">
      <string>Paste here the code you get when you login.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="currentUserLabel">
     <property name="text">
      <string notr="true">not logged in</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="loadUserButton">
     <property name="text">
      <string>load</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>UserID:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="userNameLineEdit">
     <property name="inputMask">
      <string notr="true">00000000000000000000000000000000; </string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
